### DISCUSSION TITLE
Cloud Modeling Plans and Priorities

### DISCUSSION THREAD

**Discussion by zadelman** [CMASCenter/modeling-in-the-cloud#4] [Ideas]
- What are the plans and priorities for cloud computing in your organization? 
- What would be most helpful for this workgroup to do to support you or your organization with modeling in the cloud?

**Comment by JiaoyanHuang** [comment]
We built CAMx on AWS and are planning to build CMAQ on AWS in the near future, we plan to run multiple grid sizes 36km  COUNS, 12 WUS/EUS, and 4km for the target areas.

- does the group or EPA plan to upload IWDW data to AWS?
- A general step by step github readme file to go through how to build or share libraries/SMOKE/CMAQ on AWS would be very helpful.
- One of my colleagues would like to ask how to set up AWS parallelcluster.
- It would be very helpful, if we can have information (eg. a table) of the cost for running default CMAQ/CAMx using 12 km data from IWDW.

Thanks
